bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,514 @@
+|          |signal       |Number   |yes      |WiFi signal strength (RSSI)                                                      |
+|          |lastAlarm    |String   |yes      |Most recent alarm for health check                                               |
+
+
+
+### Event Channel
+
+The binding maps the inbound HTTP events from the device into trigger events for those enabled in the Thing Configuration (see above).
+
+|Event Type|Description|
+|------------|-----------------------------------------------------------------------------------------------------------------|
+|BTN_ON      |The Button was pressed                                                                                           |
+|BTN_OFF     |The Button was released                                                                                          |
+|SHORTPUSH   |A short push to the button was detected  (requires FW 1.5.6)                                                     |
+|SHORTPUSH   |A long push to the button was detected   (requires FW 1.5.6)                                                     |
+|OUT_ON      |Relais output is on                                                                                              |
+|OUT_OFF     |Relais output is off                                                                                             |
+|ROLLER_OPEN |Roller has reached the OPEN position and stopped.                                                                |
+|ROLLER_CLOSE|Roller has reached the CLOSED position and stopped.                                                              |
+|ROLLER_SRTOP|Roller was STOPped while moving (not in the OPEN or CLOSE position)                                              |
+|ALARM       |The health monitoring detected a alarm condition, check "Alarm Events" details.                                  |
+                                                                                                                               |
+
+The channel is implemented as a Trigger and could be used in a Rule:
+```
+rule "Shelly Events"
+when
+    Channel "shelly:shelly1:XXXXXX:device#event" triggered
+then
+    logInfo("Shelly", "A Shelly device event of type "+receivedEvent.toString()+"was triggered")
+end
+```
+
+See Rule examples for information how to process the event payload.
+
+### Alarm Events
+
+The binding does some health checking on the device
+
+- The RSSI signal is monitored every 10 minutes. A alarm is send to the event channel when RSSI is < -80, which indicates an instable connection.
+- An alarm is send to the event channel when over heating, overload or a load error is reported by the device.
+- Device reports over temperature, over load, over load.
+
+Those alarms could be received through the event channel as a trigger. 
+Channel lastAlarm provides a textial description.
```

Comment:
You mean the event channel ONLY says that there is "something" and the what has to be retrieved by the "lastAlarm" channels state? If so, this again does not make sense. Users could simply react upon state changes of the "lastAlarm" channel and there is no need for the event channel for alarms.

Reply:
I turn the alarm into a trigger channel with defined alarm types, see above

Reply:
Ok, so the "lastAlarm" channel is removed then?